libflux/src/ast/mod.rs

```diff
@@ -1,4 +1,5 @@
 pub mod check;
+#[allow(warnings)]
```

Comment:
can you add a comment here why we are allowing warnings? I assume its because it is generated code?

Reply:
Sure.

Reply:
Not all of the flatbuffers module is generated, only its submodule `ast_generated`, so this could be tightened up.